modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/import/ImportForm.js

```diff
@@ -119,7 +176,6 @@ function ImportForm({
 
 		function handleFileSchemaUpdate({firstItemDetails, schema}) {
 			setFileFields(schema);
-
 			setDemoFileValues(firstItemDetails);
```

Comment:
You can handle the file content updates here.
Also, the new line made sense since it was between two code executions